src/modules/Tasks.js

```diff
@@ -0,0 +1,49 @@
+class Task {
+  constructor(description, index, completed = false) {
+    this.description = description;
+    this.index = index;
+    this.completed = completed;
+  }
+
+  static LoadList = (tasksList) => {
+    const todolist = document.querySelector('.list');
+    let string = '';
+    tasksList.forEach((element) => {
+      string += `<li class="taskContainer">
+      <input type="checkbox" class="checkbox" id="${element.index}">
+      <input type="text" id="${element.index}" class="task" value="${element.description}">
+      <button id="${element.index}" class="delete"></button>
+    </li>`;
+    });
+    todolist.innerHTML = string;
+  };
+
+  static removeObWithId = (arr, id) => {
+    const ObToRemove = arr.findIndex((obj) => obj.index === id);
+    if (ObToRemove > -1) {
+      arr.splice(ObToRemove, 1);
+    }
+    return arr;
+  };
+
+  static renewIndex = (arr) => {
+    let index = 0;
```

Comment:
- [ ] As stated in the requirements, if a 5th task is added to the list, the index of that task should equal to 5, so counting the index should start from 1 not from 0 😅 Kindly fix this issue 👍